src/components/DataViewFilterMultiselect/DataViewFilterMultiselect.styled.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+
+export const ChipWrapper = styled.div`
+	margin-top: 24px;
+	& .MuiChip-root {
+		display: flex;
+	};
+	& .MuiChip-label {
+		flex: 1 1 0%;
+	};
+`;
+
+export const StyledWrapper = styled.div`
+	padding: 22px 6px 14px;
+
+	& > .topBlock {
+		display: flex;
+		height: 400px;
+		width: 600px;
+	}
+
+	& > .topBlock > .options > .searchBar {
+		position: sticky;
+		top: 0;
+		display: flex;
+		align-items: center;
+		padding: 8px 16px;
+		z-index: 1;
+		background: ${theme.newColors.grey1[100]};
+		border: 1px solid ${theme.newColors.simplyGrey[100]};
+		margin-bottom: 24px;
+	}
+
+	& > .topBlock > .options > .searchBar > svg {
```

Comment:
Since we'll be updating our textfield to use the other one, these styles are no longer needed.